packages/headless/src/modules/user-auth/controllers/auth.controller.ts

```diff
@@ -0,0 +1,19 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { CreateUserDto, LoginDto } from '../interface/user.interface';
+import { UserAuthService } from '../services/user-auth.service';
+
+@Controller('user-auth')
```

Comment:
Is the API name okay with the old project?

Reply:
resolved